project/mkdocs.yml.jinja

```
@@ -40,7 +40,7 @@ markdown_extensions:
 - pymdownx.tabbed
 - pymdownx.tasklist
 - toc:
-    permalink: "¤"
+    permalink: True
```

Not sure if `True` is accepted. By convention we use `true` and `false` in YAML (or `yes`, `no`, `on`, `off`)

```suggestion
    permalink: true
```